src/main/java/io/icker/factions/database/PlayerConfig.java

```diff
@@ -11,29 +11,51 @@
 
     public UUID uuid;
     public ChatOption chat;
+    public boolean bypass;
 
     public static PlayerConfig get(UUID uuid) {
-        Query query = new Query("SELECT chat FROM PlayerConfig WHERE uuid = ?;")
+        Query query = new Query("SELECT * FROM PlayerConfig WHERE uuid = ?;")
             .set(uuid)
             .executeQuery();
 
-        if (!query.success) return new PlayerConfig(uuid, ChatOption.GLOBAL);
+        if (!query.success) return new PlayerConfig(uuid, ChatOption.GLOBAL, false);
 
         try {
-            return new PlayerConfig(uuid, Enum.valueOf(ChatOption.class, query.getString("chat")));
+            return new PlayerConfig(uuid, Enum.valueOf(ChatOption.class, query.getString("chat")), query.getBool("bypass"));
         } catch (IllegalArgumentException e) {
-            return new PlayerConfig(uuid, ChatOption.GLOBAL);
+            return new PlayerConfig(uuid, ChatOption.GLOBAL, false);
         }
     }
 
-    public PlayerConfig(UUID uuid, ChatOption chat) {
+    public PlayerConfig(UUID uuid, ChatOption chat, boolean bypass) {
         this.uuid = uuid;
         this.chat = chat;
+        this.bypass = bypass;
     }
 
     public void setChat(ChatOption chat) {
-        new Query("MERGE INTO PlayerConfig KEY (uuid) VALUES (?, ?);")
-            .set(uuid, chat.toString())
+        new Query("MERGE INTO PlayerConfig KEY (uuid) VALUES (?, ?, ?);")
+            .set(uuid, chat.toString(), bypass)
+            .executeUpdate();
+    }
+
+    public boolean getBypass() {
```

Comment:
You don't need this getBypass func here. Instead the attribute `bypass` should be accessible

Reply:
I had it just in case `bypass` changed, but seeing as I initialize a new PlayerConfig option every time I access it I doubt there will be a race condition that causes it to change. I'll change this.